org.adempiere.base/src/org/compiere/model/X_AD_Package_Exp_Detail.java

```diff
@@ -934,6 +950,8 @@ public String getTarget_Directory ()
 	public static final String TYPE_ImportFormat = "IMP";
 	/** Info Window = IW */
 	public static final String TYPE_InfoWindow = "IW";
+	/** Java Shell Script = JSH */
+	public static final String TYPE_JavaShellScript = "JSH";
```

Comment:
This naming choice sounds a little confusing to me since it got nothings to do with "shell". I would prefer to call it just "Script" instead.

Reply:
renamed to ScriptJSR223